jdaviz/components/viewer_data_select.vue

```diff
@@ -168,6 +168,16 @@ module.exports = {
         } else if (this.$props.viewer.reference === 'spectrum-2d-viewer') {
           return (item.ndims === 2 || item.type==='trace') && this.dataItemInViewer(item, returnExtraItems)
         }
+      } else if (this.$props.viewer.config === 'lcviz') {
+        // TODO: generalize itemIsVisible so downstream apps can provide their own customized filters
+        if (item.meta._LCVIZ_EPHEMERIS !== undefined) {
+          if (!this.$props.viewer.reference.startsWith('flux-vs-phase:')) {
+            return false
+          }
+          var viewer_comp = this.$props.viewer.reference.split('flux-vs-phase:')[1]
+          return item.meta._LCVIZ_EPHEMERIS.ephemeris == viewer_comp && this.dataItemInViewer(item, returnExtraItems)
```

Comment:
Just to make this easier to read in the future:
```suggestion
          var ephemeris_component = this.$props.viewer.reference.split('flux-vs-phase:')[1]
          return item.meta._LCVIZ_EPHEMERIS.ephemeris == ephemeris_component && this.dataItemInViewer(item, returnExtraItems)
```

Reply:
updated to `viewer_ephem_comp` (I want viewer to be in there somewhere since I think that context is important, but agree that ephemeris is important as well).